### DISCUSSION TITLE
New version of the board.

### DISCUSSION THREAD

**Discussion by hetii** [emsesp/EMS-ESP32#1413] [General]
Hello.
I almost finish my board, based on such circuit:
![obraz](https://github.com/emsesp/EMS-ESP32/assets/128046745/36322b64-9495-4cf4-8d0c-7c2ca2252edf)

The size is the same like board called goouuu-esp32 from aliexpress.
![obraz](https://github.com/emsesp/EMS-ESP32/assets/128046745/73f6782c-09b4-416e-8eac-ffdc94cf22b1)

![obraz](https://github.com/emsesp/EMS-ESP32/assets/128046745/f2898fda-ac70-49a7-b1cc-60b111ca4dcf)
![obraz](https://github.com/emsesp/EMS-ESP32/assets/128046745/2d6a2ae9-0175-44e1-89f4-34fcffebe96e)
![obraz](https://github.com/emsesp/EMS-ESP32/assets/128046745/23f23cd6-855c-40a8-87f0-643c073417cf)

![obraz](https://github.com/emsesp/EMS-ESP32/assets/128046745/e62418fc-a72f-4499-8674-58828d5dee4f)

There is single pin (2.54) 02x05 header that expose following pins:
1. 5v power supply
2. ems bus
3. 1 wire bus for ds18b20
4. water leaking sensor
5. output to control hot water circulation pomp.
On the board additionally I added two smd leds and one 3mm THT led that can be exposed outside.
Additionally there is also a piezo speaker controlled from pwm output for alarm purposes.

Due limited space I don's use fuses on ems bus and I wonder if anyone of you had any failure in this area.

This board was designed as a single layer for home producing, but I probably will fabricate in jlpcb few of them for my usage.

If anyone is interested in such version of the board, I can publish the entire project.

@bbqkees I know there was several versions of the hardware and as such I want to kindly ask if you see anything that can be improved in used circuit or maybe you see some bugs.

**Comment by refuse2speak** [comment]
@hetii would you mind sharing the project? cheers :)
